src/components/common/EdgeCreateDialog.test.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { EdgeCreateDialog } from './EdgeCreateDialog';
+import { intl } from '../../setupTests';
+import { EntityManager } from './EntityManager';
+import { Model, Entity } from '@alephdata/followthemoney';
+import { render } from '@testing-library/react';
+
+const model = new Model({
+  schemata: {
+    thing: {
+      label: 'Thing',
+      plural: 'Things',
+      extends: [],
+      schemata: [],
+      properties: {},
+    },
+  },
+  types: {},
+});
+
+const entityManager = new EntityManager({
+  model,
+  entities: [],
+});
+
+const source = new Entity(model, {
+  schema: 'thing',
+  id: 'source',
+});
```

Comment:
I switched our tests to TypeScript/TSX as well. As a consequence, TS complains if we do not pass an object into the component that matches the `EntityManager` interface.

For now, I’ve simply set up an `EntityManager` in this test file, as that was simple enough. Once we start adding more tests, we might need to extract or define mocks so we don’t do the same test setup over and over again. But for now, we can keep it simple.